airbyte-cdk/java/airbyte-cdk/core/src/main/kotlin/io/airbyte/cdk/integrations/destination/async/deser/AirbyteMessageDeserializer.kt

```diff
@@ -16,20 +18,20 @@ class DeserializationUtil {
      * * entire serialized message string when message is a valid State Message
      * * serialized AirbyteRecordMessage when message is a valid Record Message
      *
-     * @param messageString the string to deserialize
+     * @param message the string to deserialize
      * @return PartialAirbyteMessage if the message is valid, empty otherwise
      */
     fun deserializeAirbyteMessage(
-        messageString: String?,
-        dataTransformer: StreamAwareDataTransformer,
```

Comment:
:tada: love to see this

Reply:
love how the doc string didn't even match the parameters...